pkgs/applications/networking/calls/default.nix

```diff
@@ -80,7 +88,8 @@ stdenv.mkDerivation rec {
     "-Dgtk_doc=true"
   ];
 
-  doCheck = true;
+  # Tests failing?
+  doCheck = false;
```

Comment:
I haven't looked into these, but will do.

Reply:
Since it's a release candidate, you might be in time to get them fixed before the final release ;)

Reply:
Tests failing is a known issue: https://gitlab.gnome.org/GNOME/calls/-/issues/258. I've added this as a comment.